application/domain/domain.go

```diff
@@ -170,8 +170,11 @@ func readEnv() {
 }
 
 type AppError struct {
-	Code int    `json:"code"`
-	Key  string `json:"key"`
+	Code int `json:"code"`
+
+	// Don't change the value of these Key entries without making a corresponding change on the UI,
```

Comment:
Should this comment be in `errorcodes.go`? Seems like it would be a more visible reminder there.